conda-env/qd-rnaseq.yaml

```diff
@@ -0,0 +1,8 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python
```

Comment:
I would suggest adding specific versions to the requirements so the dont accidentally cause a conflict when they get updated

Reply:
Fixed in 68222d4